File: app/src/main/java/com/selflearning/chemistree/games/game1/PreGame_1_Screen.java
package com.selflearning.chemistree.games.game1;

import androidx.appcompat.app.AppCompatActivity;

import android.app.Activity;
import android.content.Context;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

import com.selflearning.chemistree.R;
import com.selflearning.chemistree.utilities.ActivityUtilities;

public class PreGame_1_Screen extends AppCompatActivity {

    Activity activity;
    Context context;

    private Button bPlayGame;
    private ImageView imageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        initVar();
        initView();
        initListener();
    }

    private void initVar(){
        activity = PreGame_1_Screen.this;
        context = activity.getApplicationContext();
    }

    private void initView(){
        setContentView(R.layout.activity_pre_game_1__screen);

        bPlayGame = findViewById(R.id.bPlayGame);
        imageView = findViewById(R.id.imageView2);
        loadImage();
    }

    private void initListener(){
        bPlayGame.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ActivityUtilities.getInstance().invokeNewActivity(activity, Game_1_Activity.class, -1, true);
            }
        });
    }

    private void loadImage(){
        Drawable d = activity.getResources().getDrawable(R.drawable.ic_benzol);
        imageView.setImageDrawable(d);

//        Picasso picasso = new Picasso.Builder(context).listener(new Picasso.Listener() {
//            @Override
//            public void onImageLoadFailed(Picasso picasso, Uri uri, Exception exception) {
//
//            }
//        }).build();
//
//            picasso.load(R.drawable.ic_benzol)
//                    .error(R.drawable.mendeleev)
//                    .into(imageView);

//        try {
//            InputStream is = getAssets().open("img/ic_benzol.xml");
//            Drawable d = Drawable.createFromStream(is, null);
//            imageView.setImageDrawable(d);
//        } catch (IOException e) {
//            return;
//        }
    }
}

File: app/src/main/java/com/selflearning/chemistree/games/game1/PreGame1Screen.kt
package com.selflearning.chemistree.games.game1

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.selflearning.chemistree.R
import com.selflearning.chemistree.databinding.ActivityPreGame1ScreenBinding
import com.selflearning.chemistree.utils.ActivityUtilities

class PreGame1Screen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityPreGame1ScreenBinding.inflate(layoutInflater)
        initVar()
        initView(binding)
        initListener(binding)
    }

    private fun initVar() {
    }

    private fun initView(binding: ActivityPreGame1ScreenBinding) {
        setContentView(binding.root)
        loadImage(binding)
    }

    private fun initListener(binding: ActivityPreGame1ScreenBinding) {
        binding.bPlayGame.setOnClickListener {
            ActivityUtilities.invokeNewActivity(this, Game_1_Activity::class.java, -1, true)
        }
    }

    private fun loadImage(binding: ActivityPreGame1ScreenBinding) {
        val d = this.resources.getDrawable(R.drawable.ic_benzol)
        binding.imageView2.setImageDrawable(d)

    }
}